addons/event/static/src/views/event_registration_kanban_controller.js

```diff
@@ -0,0 +1,58 @@
+import { kanbanView } from "@web/views/kanban/kanban_view";
+import { KanbanController } from "@web/views/kanban/kanban_controller";
+import { EventRegistrationSummaryDialog } from "@event/client_action/event_registration_summary_dialog";
+import { onMounted, onWillUnmount } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+
+export default class EventRegistrationKanbanController extends KanbanController {
+
+    setup() {
+        super.setup()
+        this.dialog = useService("dialog");
+        this.orm = useService("orm")
+        onMounted(this.hideSystray);
+        onWillUnmount(this.showSystray);
+    }
+
+    async openRecord(record, mode) {
+        if (!this.props.context.active_model) {
+            const barcode = record.data.barcode
+            const eventId = record.data.event_id[0]
+
+            const result = await this.orm.call("event.registration", "register_attendee", [], {
+                barcode: barcode,
+                event_id: eventId,
+            });
+
+            this.dialog.add(
+                EventRegistrationSummaryDialog,
+                {
+                    registration: result,
+                    model: this.model
+                }
+            )
+        } else {
+            return super.openRecord(record, mode);
+        }
+    }
+
+    hideSystray() {
+        this.observer = new MutationObserver((mutations) => {
+            document.querySelector('.o_menu_systray').classList.add('d-none');
+            this.observer.disconnect();
+        });
```

Comment:
No, systray is not present when `onMounted` is called. I think it is added after the setup is called (as it is not part of the kanban view). I tried but didn't find any solution other than `setTimeout` :sweat_smile: 
`MutationObserver` works here because it looks for the change in DOM and as soon as systray is loaded it hides it and then observer disconnects.
I am open for the ideas though :face_with_peeking_eye: 
Thanks.